app/src/main/java/com/wizeline/academy/animations/ui/more_details/MoreDetailsFragment.kt

```diff
@@ -29,6 +38,17 @@ class MoreDetailsFragment : Fragment() {
     ): View {
         _binding = MoreDetailsFragmentBinding.inflate(inflater, container, false)
         binding.ivImageDetailLarge.loadImage(args.imageId)
+
+        val title = binding.tvTitle;
+        val imageDetail = binding.ivImageDetailLarge;
+        val fullText = binding.tvFullTextContent;
+
+        val sideRightSlideAnimation = AnimationUtils.loadAnimation(this.context, R.anim.side_right_slide_animation)
```

Comment:
R.anim means View.animation API which is the oldest, it is the oldest one and the only use case is the window transition, instead I recommend ObjectAnimator or ViewPropertyAnimator, example:

 title.animate()
            .alpha(1f)
            .start()

also this transition was requested as shared element transition, you can follow this link:
https://developer.android.com/guide/navigation/navigation-animate-transitions#shared-element